packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -796,6 +798,7 @@ export default class CommonmarkRenderer extends Renderer {
           headerText = name;
         }
       }
+      name = name === "" ? `column ${index + 1}` : name;
```

Comment:
Doesn't this mean we are coercing empty table headers to have this value? That is, as a user, if I create a table with an empty header in rich text, doesn't this turn it into "column X" after I save?